### ISSUE TITLE
8.4.0 seems to break some functionality

### ISSUE THREAD

**Issue by solipsist01** [coderaiser/cloudcmd#150]
Hello :)

I updated immediately with the new functionality to 8.4.0 :)
I have to say that changing the layout works perfectly and i want to thank you for that.

Though since the update to 8.4.0 i have some weird behaviour.

I run cloudcmd in a docker container this way

`docker run --name=cloudcmd -d -v ~:/root -v /:/mnt/fs -p 8088:8000 coderaiser/cloudcmd:latest-alpine --root /mnt/fs/sharedfolders/ --auth -u redactedusername -p 'redactedpassword'
`

Copy, Move, Delete (f5, f6, f8) don't seem to do anything anymore
When i try to rename a file (f2), the container exits with this error:

```
/usr/src/app/server/rest.js:308
    let names = files.names.slice();


TypeError: Cannot read property 'slice' of undefined
    at moveFiles (/usr/src/app/server/rest.js:308:29)
    at onPUT (/usr/src/app/server/rest.js:185:9)
    at pullout (/usr/src/app/server/rest.js:102:13)
    at IncomingMessage.onEnd (/usr/src/app/node_modules/pullout/legacy/pullout.js:36:9)
    at emitNone (events.js:106:13)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

to be sure it's not my setup, i have pulled the previous image.

`docker run --name=cloudcmd -d -v ~:/root -v /:/mnt/fs -p 8088:8000 coderaiser/cloudcmd:8.2.2-alpine --root /mnt/fs/sharedfolders/ --auth -u redactedusername -p 'redactedpassword'`

This image doesn't have this problem :)

**Comment by kirtapix** [comment]
Same problem here, I reverted back on 8.3.0.

By the way, @coderaiser, your work is amazing.

**Comment by coderaiser** [comment]
Thank you :), fixed with 0645e91. Landed in [v8.4.1](https://github.com/coderaiser/cloudcmd/releases/tag/v8.4.1).
